test/closure.ts

```diff
@@ -0,0 +1,59 @@
+import * as child_process from 'child_process';
+
+export type Flags = {
+  [flag: string]: boolean|string|string[]
+};
+
+/** The type of compilation results, containing exit code and output. */
+export interface Result {
+  exitCode: number;
+  stdout: string;
+  stderr: string;
+}
+
+export class Compiler {
```

Comment:
does this class pull its own weight, or should it just be a function?

Reply:
I was originally trying to match to original API, then I diverged, so I may as well go all the way.  Done.